_docs_use-cases/use-case-consumption-get-repositoryattributes.md

```diff
@@ -0,0 +1,60 @@
+---
+# Start automatic generation
+permalink: use-case-consumption-get-repositoryattributes
+published: true
+title: "Get RepositoryAttributes"
+type: use-case
+toc: true
+sidebar:
+  - title: "Integrate Enmeshed"
+    nav: "docs_integrate"
+properties:
+  - id: RA22
+  - component: Runtime
+  - layer: Consumption
+  - facade: AttributesFacade
+  - function: getRepositoryAttributes
+  - description: Fetches all RepositoryAttributes
+  - feature category: Normalized attributes
+  - tech category: Attributes
+  - status: DONE
+  - documentation status: DONE
+  - comments:
+  - actor: Identity
+  - trigger: REST API
+  - precondition:
+  - result:
+  - priority:
+  - complexity:
+  - size:
+  - created_at:
+  - changed_at:
+  - api_route_regex: GET /api/v2/Attributes/Own/Repository
```

Comment:
I suppose it would be nice to adjust the route name too, but unfortunately backwards incompatible? Maybe we should put this on a TODO list.

Reply:
I think the name of the route actually isn't that old. For me it is fine. Otherwise, talk to @sebbi08 .